src/System.Diagnostics.FileVersionInfo/System.Diagnostics.FileVersionInfo.sln

```diff
@@ -25,6 +28,12 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "src", "src", "{E107E9C1-E89
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "ref", "ref", "{2E666815-2EDB-464B-9DF6-380BF4789AD4}"
 EndProject
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "NativeConsoleApp", "tests\NativeConsoleApp\NativeConsoleApp.vcxproj", "{3002560A-5097-4A65-B562-93E335E64163}"
```

Comment:
Do we want these in the solution? I would lean towards not having them there so that folks working on this library don't have to worry about these at all.